### DISCUSSION TITLE
How can I configure specific rule sets through CLI options?

### DISCUSSION THREAD

**Discussion by andyluooo** [astral-sh/ruff#4254] [Q&A] [answered]
For example, I am currently using the `D` rules (implemented from pydocstyle) and want to enable the `google` style. I am curious that if there's a way I can configure such non-top level things through CLI options without a configuration file?

**Comment by charliermarsh** [comment, marked as answer]
Ah yeah, unfortunately, there's no way to set those non-top-level settings through the CLI right now.
